paperqa/types.py

```diff
@@ -313,16 +312,18 @@ def reduce_content(self) -> str:
 
 
 class DocDetails(Doc):
-    model_config = ConfigDict(validate_assignment=True, extra="ignore")
+    model_config = ConfigDict(validate_assignment=True)
```

Comment:
```suggestion
    model_config = ConfigDict(validate_assignment=True, extra="ignore")
```

Actually, let's just revert this `DocDetails.model_config` change

Reply:
Isn't `extra="ignore"`default?